app/views/provider_interface/offers/show.html.erb

```diff
@@ -39,6 +31,21 @@
             border: false,
             ske_conditions: @application_choice.offer.ske_conditions,
           ) %>
+
+    <% elsif @provider_user_can_make_decisions && @application_choice.pending_conditions? %>
+      <%= render ProviderInterface::ChangeCourseOfferSummaryComponent.new(
+        application_choice: @application_choice,
+        course_option: @application_choice.current_course_option,
+        conditions: @conditions,
+        course: @application_choice.current_course,
+        available_providers: @providers,
+        available_courses: @courses,
+        available_course_options: @course_options,
+        border: false,
+        editable: @provider_user_can_make_decisions,
+        ske_conditions: @application_choice.offer.ske_conditions,
+        show_recruit_pending_button: true,
```

Comment:
It's a shame this component takes so many kwargs that could just be derived from the application_choice, but it's beyond the scope of this ticket to address that.